Packs/Core/Integrations/CortexCoreIR/CortexCoreIR.yml

```diff
@@ -1666,9 +1666,20 @@ script:
     - description: Comma-separated list of endpoint IDs. Can be retrieved by running the core-get-endpoints command.
       name: endpoint_ids
       required: true
-    - description: Comma-separated list of shell commands to execute.
+    - description: Comma-separated list of shell commands to execute. Use is_raw_command=false to prevent splitting by commas.
       name: commands
       required: true
+    - description: Determines if shell commands are passed in their raw form or split by commas.
```

Comment:
```suggestion
    - description: Whether to pass the command as-is. When false, the command is split by commas and sent as a list of commands, that are run independently.
```

Reply:
Fixed in the YML and REAME in commit https://github.com/demisto/content/commit/df4e4271a26914a697650c106fa327f222428475